aten/src/ATen/native/cpu/TensorCompareKernel.cpp

```diff
@@ -330,7 +330,7 @@ static void isin_default_kernel_cpu(
 }
 
 static void clamp_kernel_impl(TensorIteratorBase& iter) {
-  AT_DISPATCH_ALL_TYPES_AND(kBFloat16, iter.common_dtype(), "clamp_cpu", [&]() {
+  AT_DISPATCH_ALL_TYPES_AND3(kHalf, kBFloat16, kBool, iter.common_dtype(), "clamp_cpu", [&]() {
```

Comment:
I'm not in love with enabling Half and Bool (we are trying to reduce or not extend half support on CPU, and Bool makes little sense for clamp/min/max and often results in unexpected type promotion, like `torch.minimum(x, 0)` would produce integer tensor from bool input `x`, but if it simplifies other things, that's fine.

Reply:
This was the easiest way to fix tests. I'll look for alternative ways to fix them and let you know.

Reply:
OpInfo tests allow you to specify supported dtypes differently per device, so you might just change that. But again, if that's too time consuming or doesn't work, we can go with this expanded functionality.

Reply:
Done, now I don't touch the regular clamp dtypes